Fix FTBFS in VirtualBox 3.0.14 OSE (taken from 3.1.4 OSE)

diff -ur a/src/VBox/Frontends/VirtualBox/src/VBoxAboutDlg.cpp b/src/VBox/Frontends/VirtualBox/src/VBoxAboutDlg.cpp
--- a/src/VBox/Frontends/VirtualBox/src/VBoxAboutDlg.cpp	2010-03-18 11:30:45.000000000 -0400
+++ b/src/VBox/Frontends/VirtualBox/src/VBoxAboutDlg.cpp	2010-03-25 14:51:46.051578843 -0400
@@ -24,6 +24,7 @@
 #include "VBoxGlobal.h"
 
 #include <iprt/path.h>
+#include <VBox/version.h> /* VBOX_VENDOR */
 
 /* Qt includes */
 #include <QDir>
